articles/logs/streams/azure-event-grid.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Azure Event Grid Log Streams
+description: Learn how to create an event-driven workflow using Azure Event Grid and send your tenant logs anywhere within the Azure ecosystem.
+toc: false
+topics:
+ - logs
+ - streams
+ - event-streams
+contentType: how-to
+---
+
+# Integrate Azure Event Grid with Auth0
+
+Azure Event Grid is a serverless event bus that lets you send event data from any source to any destination.
+
+You can create event-driven workflows using to Event Grid send your Auth0 tenant logs to targets such as Azure Functions, Event Hubs, Sentinel, and Logic Apps.
+
+See the Auth0 event type codes for a [full list](https://auth0.com/docs/logs/references/log-event-type-codes) of the events that Auth0 supports.
+
+## Send events from Auth0 to Azure Event Grid
+
+To send Auth0 events to Azure, you'll need to:
+
+1. Enable Event Grid resource provider.
+2. Set up an event source (in this case, this is Auth0).
+3. Set up an event handler, the app or service where the event will be sent.
+
+For more information about these concepts, [see Concepts in Azure Event Grid](https://docs.microsoft.com/en-us/azure/event-grid/concepts)
+
+### Enable Event Grid resource provider
+
+If you haven’t previously used Event Grid, you will need to register the Event Grid resource provider. If you've used Event Grid before, skip to the next section.
+
+In your Azure portal:
+
+1. Select Subscriptions on the left menu
```

Comment:
I'm not sure Subscriptions is guaranteed to be on the left nav, because I believe the items on there are dynamic based on what you've used recent. E.g. my left nav doesn't have it:

<img width="1642" alt="Screen Shot 2020-05-15 at 10 37 02" src="https://user-images.githubusercontent.com/48957/82079741-3369f980-9698-11ea-842f-6c45af81676c.png">